client/jetpack-onboarding/steps/summary.jsx

```diff
@@ -45,32 +46,36 @@ class JetpackOnboardingSummaryStep extends React.PureComponent {
 				/>
 				<FormattedHeader headerText={ headerText } subHeaderText={ subHeaderText } />
 
-				<div className="steps__summary-columns">
-					<div className="steps__summary-column">
-						<h3 className="steps__summary-heading">{ translate( "Steps you've completed:" ) }</h3>
-						<CompletedSteps
-							basePath={ basePath }
-							onClick={ this.handleSummaryStepClick }
-							siteId={ siteId }
-							siteSlug={ siteSlug }
-							steps={ steps }
-						/>
+				<Card>
+					<div className="steps__summary-columns">
+						<div className="steps__summary-column">
```

Comment:
Now that we have some additional spacing coming from the `<Card />`, the column contents might look a little tight on some smaller devices:

![](https://cldup.com/y5_F65Fcjm.png)

We should be able to fix this by adding something like:

```
.steps__summary-column {
	@include breakpoint( '<660px' ) {
		margin-left: 30px;
		margin-right: 30px;
	}
```

which should also properly apply gracefully to all smaller devices.